types/scrollmagic/index.d.ts

```diff
@@ -0,0 +1,195 @@
+// Type definitions for scrollmagic 2.0
+// Project: https://scrollmagic.io/
+// Definitions by: Tim Weise <https://github.com/switchnollie>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/* Scrollmagic is a UMD module that exposes a global variable ScrollMagic
+when loaded outside a module loader */
+export default ScrollMagic;
```

Comment:
This might be an error but I'm not very familiar with the library, I don't think it exports a default. 

Could you please double check?

Reply:
In CJS the scrollmagic bundle exports a single namespace function i.e.
```js
function ScrollMagic() {/*...*/}
ScrollMagic.Controller = function() {/**/}
ScrollMagic.Scene = function() {/**/}
module.exports = ScrollMagic
```
My understanding is that this is equivalent to a default export in ESM as there is no concept of a default export in CJS. 
To import the namespace in a ESM environment, both a default import (`import ScrollMagic from "scrollmagic"`) and a module object (`import * as ScrollMagic from "scrollmagic"`) are working, so 
e.g.
```js
import ScrollMagic from "scrollmagic";

console.log({ ScrollMagic });
/**
 <- ScrollMagic: ƒ ()
      ...
      version: "2.0.7"
      Controller: ƒ (options)
      Scene: ƒ (options)
      ...
    __proto__: Object
 */
```
I used the default export in TS so that the types/interfaces can be imported as named imports and there is a distinction between the library (default) and types (named).

Reply:
I still think this is incorrect as still the library doesn't export a `default`. Instead you should take a look at the `esModuleInterop` flag, which I think is used for scenarios like the one you mentioned,

Reply:
Now I get it thank you for the advice! I added the `esModuleInterop` and `allowSyntheticDefaultImports` and now it's working as expected without the default export.

Reply:
Any news? I need the types for a library that I am working on

Reply:
Merged. Sorry for the delay, seems like I missed the notification in the email hell.